assets/js/utils/store.ts

```diff
@@ -6,7 +6,7 @@ export const lastUpdatedSuffix = '__lastUpdated';
 
 export default {
 
-  set(key, value) {
+  set(key: string, value: unknown) {
```

Comment:
Perhaps `set<Value = unknown>(key: string, value: Value)`, and same with others below? I assume this was only done to avoid invasively modifying other code

Reply:
Since the type of `value` is not important at any future point, leaving it as unknown is enough